courseraresearchexports/exports/utils.py

```diff
@@ -50,16 +50,17 @@ def download_export(export_job, dest):
     Download an export archive if ready given an export_job_id
     :param export_job:
     :param dest:
-    :return:
+    :return output_filename:
     """
-    if export_job['exportType'] == 'RESEARCH_WITH_EVENTING':
+    if export_job['exportType'] == EXPORT_TYPE_EVENTING:
         logging.error('Generate download links to download eventing exports')
-        raise ValueError('Require RESEARCH_WITH_SCHEMAS job')
+        raise ValueError('Require {} job'.format(EXPORT_TYPE_SCHEMAS))
     elif 'downloadLink' not in export_job:
         logging.error('Export job {} is not ready'.format(export_job['id']))
         raise ValueError('Export job is not yet ready for download')
```

Comment:
can we make this more informative? (e.g the job is IN_PROGRESS, please wait or TERMINATED, please contact data-support@coursera.org if we do not resolve this in the next 24 hours)